TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java

```diff
@@ -12,13 +12,13 @@
 	public AmazonSignIn(WebDriver driver) {
 		this.driver = driver;
 	}
-	public void InputEmail(String email) {
+	public void inputEmail(String email) {
 		driver.findElement(By.xpath("//input[@id='ap_email']")).sendKeys(email);

```

Comment:
WebElemenst declaration move out from the methods and paste it below or above public AmazonSignIn(WebDriver driver) and should be done like this @FindBy(xpath="//input[@id='advancedFinder']")
	private WebElement advanceFinderToogle;